crates/rt/src/thread.rs

```diff
@@ -1,7 +1,6 @@
 use core::ffi::CStr;
-use core::ptr;
 
-use flipperzero_sys as sys;
+use flipperzero_sys::{self as sys};
```

Comment:
This looks like a spurious diff, the previous variant is the preferred one

Reply:
For some reason this was automatically created, I thought I removed it. Thanks for pointing out!